Makefile

```diff
@@ -195,14 +196,15 @@ maintainer-conf: dist composer-dependencies-dev
 
 # Run Symfony in dev mode (for maintainer-mode):
 webapp/.env.local:
-	echo "# This file was automatically created by 'make maintainer-install' to run" >> $@
+	echo "# This file was automatically created by 'make maintainer-conf' to run" >> $@
```

Comment:
This was already wrong, but the first line should probably use `>` to make sure the file is recreated from scratch instead of appended to.